.pre-commit-config.yaml

```diff
@@ -722,50 +737,17 @@ repos:
         pass_filenames: false
         args:
           - -v
+          - --build-isolation
           - --py-version=3.10
           - --include=requirements/base.txt
           - --include=requirements/zeromq.txt
           - --include=requirements/pytest.txt
           - --include=requirements/static/pkg/linux.in
           - --include=requirements/static/ci/common.in
-          - --pip-args=--constraint=requirements/static/ci/py{py_version}/linux.txt
           - requirements/static/ci/cloud.in
   # <---- Cloud CI Requirements --------------------------------------------------------------------------------------
 
   # ----- Doc CI Requirements --------------------------------------------------------------------------------------->
-##      - id: pip-tools-compile
-##        alias: compile-doc-requirements
-##        name: Docs CI Py3.6 Requirements
-##        files: ^requirements/((base|zeromq|pytest)\.txt|static/ci/(docs|common|linux)\.in|static/pkg/linux\.in|static/pkg/.*/linux\.txt)$
-##        pass_filenames: false
-##        args:
-##          - -v
-##          - --py-version=3.6
-##          - --platform=linux
-##          - --include=requirements/base.txt
-##          - --include=requirements/zeromq.txt
-##          - --include=requirements/static/pkg/linux.in
-##          - --include=requirements/static/ci/linux.in
-##          - --include=requirements/static/ci/common.in
-##          - --pip-args=--constraint=requirements/static/ci/py{py_version}/linux.txt
-##          - requirements/static/ci/docs.in
-
-      - id: pip-tools-compile
-        alias: compile-doc-requirements
```

Comment:
Why are we removing 3.7 ?

Reply:
I had troubles with the dependencies conflicting and since we build docs on 3.8 O remeved them.

Anyway, this is not done, and will likely need to be redone. On some local changes, I added back these requirements files.

Reply:
Added back